testing/solomachine.go

```diff
@@ -171,6 +173,47 @@ func (solo *Solomachine) ConnOpenAck(chain *TestChain, clientID, connectionID st
 	require.NotNil(solo.t, res)
 }
 
+// ChanOpenInit initializes a channel on the provided chain given a solo machine connectionID.
+func (solo *Solomachine) ChanOpenInit(chain *TestChain, connectionID string) string {
+	msgChanOpenInit := channeltypes.NewMsgChannelOpenInit(
+		mock.PortID,
+		mock.Version,
+		channeltypes.UNORDERED,
+		[]string{connectionID},
+		mock.PortID,
```

Comment:
just to make sure I understand correctly, with solomachine the port and counterparty port should always be the same?

Reply:
No, this doesn't really have anything to do with solomachine per se. This is creating a channel for the "mock" model and therefore both channel ends are using the `mock` port ID.